packs/data/equipment-effects.db/effect-choker-arm-mutagen-greater.json

```diff
@@ -0,0 +1,88 @@
+{
+    "_id": "HlUL7FmGLyS35Jpp",
+    "img": "systems/pf2e/icons/default-icons/effect.svg",
+    "name": "Effect: Choker-Arm Mutagen (Greater)",
+    "system": {
+        "description": {
+            "value": "<p>Granted by @UUID[Compendium.pf2e.equipment-srd.Choker-Arm Mutagen (Greater)]{Choker-Arm Mutagen (Greater)}</p>\n<p><strong>Implemented Effects</strong></p>\n<ul>\n<li>+10 foot reach</li>\n<li>+3 item bonus to Acrobatics checks to @UUID[Compendium.pf2e.actionspf2e.Escape]{Escape}, @UUID[Compendium.pf2e.actionspf2e.Squeeze]{Squeeze}, and @UUID[Compendium.pf2e.actionspf2e.Tumble Through]{Tumble Through}</li>\n<li>-1 penalty to Athletics checks, Stealth checks, Thievery checks, and attack rolls</li>\n<li>-1 penalty per damage die to all weapon and unarmed attack damage.</li>\n</ul>"
+        },
+        "duration": {
+            "expiry": "turn-start",
+            "sustained": false,
+            "unit": "hours",
+            "value": 1
+        },
+        "level": {
+            "value": 1
+        },
+        "rules": [
+            {
+                "key": "ActiveEffectLike",
+                "mode": "add",
+                "path": "system.attributes.reach.base",
+                "value": 10
+            },
+            {
+                "key": "ActiveEffectLike",
+                "mode": "add",
+                "path": "system.attributes.reach.manipulate",
+                "value": 10
+            },
+            {
+                "key": "FlatModifier",
+                "predicate": [
+                    {
+                        "or": [
+                            "action:escape",
+                            "action:squeeze",
+                            "action:tumble-through"
+                        ]
+                    }
+                ],
+                "selector": "acrobatics",
+                "type": "item",
+                "value": 3
+            },
+            {
+                "key": "FlatModifier",
+                "selector": "athletics",
+                "value": -1
+            },
+            {
+                "key": "FlatModifier",
+                "selector": "stealth",
+                "value": -1
+            },
+            {
+                "key": "FlatModifier",
+                "selector": "thievery",
+                "value": -1
+            },
+            {
+                "key": "FlatModifier",
+                "selector": "attack-roll",
+                "value": -1
+            },
+            {
+                "key": "FlatModifier",
```

Comment:
Might need a predicate?
```
"predicate": [
	{
		"or": [
			"item:type:weapon",
			"unarmed"
		]
	}
]
```

Reply:
Should be implicit with "strike-damage", no? I figure the "strike-damage" selector should only work for strikes, which would only happen with weapon and unarmed attacks. Unless I'm missing a weird edge case?

Reply:
It doesn't appear to need a predicate